migrations.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
What's the purpose of this file?

Reply:
should delete and ill add this to gitignore, its a file nx generates about what the migration is, and then you run-migrations with this file